### ISSUE TITLE
Installation fails due to missing swift-build binary

### ISSUE THREAD

**Issue by ntpeters** [neonichu/Azkaban#6]
OSX: 10.11.5
Xcode: 7.3.1

When attempting to install via `brew`, I get the following error:

```
error: unable to invoke subcommand: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-build (No such file or directory)
```

It looks like `swift-build` is no longer included in the installation for Xcode 7.3, per discussion here: [swift-build missing in xcode 7.3?](https://lists.swift.org/pipermail/swift-build-dev/Week-of-Mon-20160321/000362.html)

**Comment by neonichu** [comment]
I don't think it was ever included, only the snapshots from swift.org have it (but by now only some of them, because it was removed later in the release process). When originally developing this, Swift 2.2 implied this.

I'm going to update this to Swift 3 soon and this ambiguity should be gone again, because all variants of that should ship with the Swift Package Manager.
